### PR TITLE
MM-21229 - Customization > Site Name help text doesn't match text field behavior

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
Allow empty sitename, since it has a sane default

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-21229

### CODE REVIEW

**Comment by jespino** [comment]
This looks like is based on not related revert, is it ok?

**Comment by reflog** [comment]
@jespino - crap. sorry. will resubmit proper PR now

**Comment by reflog** [comment]
@jespino done. please re-review :)

File: i18n/de.json [hunk 1]
```diff
@@ -4457,11 +4457,7 @@
   {
     "id": "model.config.is_valid.site_url_email_batching.app_error",
     "translation": "E-Mail-Stapelverarbeitung lässt sich nicht aktivieren, wenn SiteURL nicht gesetzt ist."
-  },
```
Comment (jespino) [inline i18n/de.json]: You should only change the `en.json` file, the rest of the translations files are handled by the translation process. Please, revert all of them except the `en.json`.

**Pull request merged**